apisix/schema_def.lua

```diff
@@ -952,6 +952,11 @@ _M.plugin_injected_schema = {
                     { type = "object" },
                 }
             },
+            priority = {
+                description = "priority of plugins by customized order",
+                type = "integer",
+                default = 0,
```

Comment:
We should not use a default value here (it should depend on the plugin)

Reply:
remove it